src/expr/skolem_manager.cpp

```diff
@@ -615,4 +615,79 @@ TypeNode SkolemManager::getTypeFor(SkolemId id,
   return TypeNode();
 }
 
+size_t SkolemManager::getNumIndicesForSkolemId(SkolemId id) const
+{
+  size_t size = 0;
+
+  switch (id)
+  {
+    // Number of skolem indices: 0
+    case SkolemId::BV_EMPTY: size = 0; break;
```

Comment:
Can you group these into common cases? And then use `return n;` instead of `size = n`.